contrib/scalapblib/src/mill/contrib/scalapblib/ScalaPBModule.scala

```diff
@@ -104,7 +104,10 @@ trait ScalaPBModule extends ScalaModule {
             case Some(entry) =>
               if (entry.getName.endsWith(".proto")) {
                 val protoDest = dest / os.SubPath(entry.getName)
-                Using(os.write.outputStream(protoDest, createFolders = true))(IO.stream(zip, _))
+                Using(os.write.over.outputStream(protoDest, createFolders = true))(IO.stream(
+                  zip,
+                  _
+                ))
```

Comment:
Two issues here: first the formatting was ugly ;) , second, `Using.apply` returns a `Try` which may hide any issues, hence I added the `resource` which will let bubble up any exception that may occur.

```suggestion
                Using.resource(os.write.over.outputStream(protoDest, createFolders = true)) { os =>
                  IO.stream(zip, os)
                }```

Reply:
My original commit basically only adds a `.over` after the `os..write` call (https://github.com/com-lihaoyi/mill/pull/2800/commits/77be6cd0f8f4267039b094cae820af6a3f051e40). I ran scalafmt and blindly committed after seeing CI fail, but I agree that the format is ugly.